core/local/analysis.js

```diff
@@ -53,7 +53,11 @@ function analyseEvents (events /*: LocalEvent[] */, pendingChanges /*: LocalChan
   const getChangeByPath = (e) => {
     return changesByPath.get(e.path)
   }
-  const changeFound = (c /*: LocalChange */) => {
+  const changeFound = (c /*: LocalChange | true */) => {
+    if (c === true) {
+      // A previous change with was transformed. Nothing more to index.
```

Comment:
What's supposed to go between `with` and `was`?

Reply:
Oops typo!

Reply:
(fixed)